pkg/config/config_test.go

```diff
@@ -417,14 +417,6 @@ func TestNewConfigMissingParameter(t *testing.T) {
 			},
 			"*backends.AWSSecretsManager",
 		},
-		{
-			map[string]interface{}{
```

Comment:
There should still be a config test for azure key vault, might look different but we don’t want to remove it